packages/@expo/cli/src/start/interface/interactiveActions.ts

```diff
@@ -40,9 +40,16 @@ export class DevServerManagerActions {
           );
         }
         Log.log(printItem(chalk`Metro waiting on {underline ${nativeRuntimeUrl}}`));
-        // TODO: if development build, change this message!
-        Log.log(printItem('Scan the QR code above with Expo Go (Android) or the Camera app (iOS)'));
+        if (options.devClient === false) {
+          // TODO: if development build, change this message!
+          Log.log(
+            printItem('Scan the QR code above with Expo Go (Android) or the Camera app (iOS)')
+          );
+        } else {
+          Log.log(printItem('Scan the QR code above to launch the custom build'));
```

Comment:
```suggestion
            Log.log(printItem('Scan the QR code above to open the project in your development build. Learn more: https://expo.fyi/start'))
```

but use `learnMore` instead of actual learn more thing like above

Reply:
also if you accept this i will create this fyi page